content/tokio/tutorial/non-async.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Non-async projects"
+---
+
+In the examples we have seen so far, we marked the main function with
+`#[tokio::main]` and made the entire project asynchronous. However you don't
+have to do this to use Tokio in your project. This page explains how you can
+isolate async/await to a small part of your project.
+
```

Comment:
It might be helpful to start by explaining some scenarios in which the user will need to avoid `#[tokio::main]`, this will help give them context.